Projects/App/Sources/UI/Organization/VerifyCode/View/VerifingCodeViewController.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  VerifingCodeViewController.swift
+//  App
+//
+//  Created by 김태호 on 2022/11/02.
+//  Copyright (c) 2022 com.zesty. All rights reserved.
+//
+
+import Combine
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class VerifingCodeViewController: UIViewController {
+    
+    // MARK: - Properties
+    private var cancelBag = Set<AnyCancellable>()
+    
+    private let isSE: Bool = UIScreen.main.isHeightLessThan670pt
+    
+    private var keyboardEndFrameHeight: CGFloat?
+    
+    private lazy var titleView = MainTitleView(title: "이메일로 받은 코드를\n알려주세요", subtitle: "\(userEmail)", hasSymbol: true)
+    
+    private let warningMessage = UILabel()
+    private let otpStackView = OTPStackView()
+    private let timerLabel = UILabel()
+    
+    private let resendStackView = UIStackView()
+    private let resendLabel = UILabel()
+    private let resendEamilButton = UIButton(type: .custom)
+    
+    private let arrowButton = ArrowButton(initialDisable: false)
+    
+    // TODO: ViewModel로 옮길 것들입니다
+    let userEmail = "mingming@pos.idserve.net"
+    var isArrowButtonHidden: Bool = true
+    var isCodeValid: Bool = true
+    var userInputCode: String = ""
+    var timer = "03:00"
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+        bindUI()
+    }
+    
+    // MARK: - Function
+    
+    @objc func resendButtonTapped() {
+        print("버튼 눌림")
+        showToastMessage()
+    }
+    
+    private func showToastMessage() {
+        let toastLabel = UILabel()
+        toastLabel.backgroundColor = .black
+        toastLabel.textColor = .white
+        toastLabel.font = .systemFont(ofSize: 16.0, weight: .regular)
+        toastLabel.textAlignment = .center
+        toastLabel.text = "메일을  다시 보냈어요"
```

Comment:
p1;

'메일을'과 '다시' 사이에 공백이 두 개나 있어요😈

```suggestion
        toastLabel.text = "메일을 다시 보냈어요"
```

Reply:
아보 정말 앙마네요

Reply:
수정했습니다! fc0fc403c24a0bf12811d8733c129b81b2004e36